tests/Maker/FunctionalTest.php

```diff
@@ -387,6 +395,94 @@ function (string $output, string $directory) {
                 ),
         ];
 
+        $makerTestAuthenticatorLoginFormUserEntity = MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeAuthenticator::class),
+            [
+                // authenticator type => login-form
+                1,
+                // class name
+                'AppCustomAuthenticator',
+                // controller name
+                'SecurityController',
+            ]
+        )
+            ->addExtraDependencies('doctrine')
+            ->addExtraDependencies('security')
+            ->addExtraDependencies('twig')
+            ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeAuthenticatorLoginFormUserEntity')
+            ->configureDatabase()
+            ->updateSchemaAfterCommand()
+            ->assert(
+                function (string $output, string $directory) {
+                    $this->assertContains('Success', $output);
+
+                    $fs = new Filesystem();
+                    $this->assertTrue($fs->exists(sprintf('%s/src/Controller/SecurityController.php', $directory)));
+                    $this->assertTrue($fs->exists(sprintf('%s/templates/security/login.html.twig', $directory)));
+                    $this->assertTrue($fs->exists(sprintf('%s/src/Security/AppCustomAuthenticator.php', $directory)));
+                }
+            );
+
+        if (Kernel::VERSION_ID < 40100) {
+            $makerTestAuthenticatorLoginFormUserEntity->addExtraDependencies('symfony/form');
```

Comment:
I think we could instead add this into the Maker. Then, the user would also see this dependency error. And, I think the tests would also automatically add it, as it WOULD be a dependency when the user is running 4.0.